compute/instances/how-to/migrate-instances.mdx

```diff
@@ -15,6 +15,8 @@ categories:
 
 The Scaleway platform makes it very easy to migrate your data from one Instance to another, or upgrade your Instance to a more powerful one if your requirements grow. In this how-to, we will upgrade an Instance by migrating from a [DEV1-S](/compute/instances/concepts/#development-instance) Instance to a [GP1-XS](/compute/instances/concepts/#general-purpose-instance) Instance. The new GP1-XS Instance will have the same [flexible IP](/compute/instances/concepts/#flexible-ip) as the original DEV1-S Instance.
```

Comment:
Do we keep tag named by old ranged ?

Reply:
From what I understood these names aren't changing yet?